src/fides/api/models/privacy_request.py

```diff
@@ -1459,7 +1459,9 @@ def hash_single_value(value: Union[str, int]) -> str:
             return hashed_value
 
         if isinstance(value, list):
-            return [hash_single_value(item) for item in value]
+            # Skip hashing lists: this avoids us hashing and later indexing potentially large values and our index
+            # is not useful for array search anyway
+            return None
```

Comment:
Here's the main change, no longer hashing lists, and I skip calling this method for search now if the value is an array